docs/CHANGELOG.rst

```diff
@@ -4,6 +4,17 @@ Change Log
 
 All notable changes to this project are documented in this file.
 
+==========
+Unreleased
+==========
+
+Added
+-----
+- Tests for annotation parsing
```

Comment:
Can you just say "Add DescriptorSheet object". Readers of this file are imaps package users - thy don't care if there are tests as long as everything works. Tests are only for your future self and other developers. :-)